src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -16,6 +15,7 @@
 
 using DynamoUnits;
 using DynamoUtilities;
+using Greg;
```

Comment:
Greg is a really unexplanatory name. As this is an introduction into the core part of the application, please can you put a line comment that states what this is

using Greg; // The Auth provider for the package manager

Alternatively we could alias the namespace?